python/cudf/cudf/pandas/__main__.py

```diff
@@ -75,6 +75,16 @@ def main():
     rmm_mode = install()
     with profile(args.profile, args.line_profile, args.args[0]) as fn:
         args.args[0] = fn
+        if "managed" in rmm_mode:
```

Comment:
I'd like to put this outside the context manager, just below `rmm_mode = install()`.

Reply:
Done 👍